api/internal/route.go

```diff
@@ -49,9 +49,9 @@ func SetUpRouter() *gin.Engine {
 	store := cookie.NewStore([]byte("secret"))
 	r.Use(sessions.Sessions("session", store))
 	r.Use(filter.CORS(), filter.RequestId(), filter.RequestLogHandler(logger), filter.SchemaCheck(), filter.Authentication(), filter.RecoverHandler())
-	r.Use(static.Serve("/", static.LocalFile(conf.WebDir, false)))
+	r.Use(static.Serve("/", static.LocalFile(conf.WorkDir + conf.WebDir, false)))
```

Comment:
It's better to use `filepath.Join` here.

Reply:
ok, I will adjust it later.
 Thanks !

Reply:
@tokers  Done, please have a look~